integration/airflow/openlineage/airflow/extractors/bash_extractor.py

```diff
@@ -17,20 +17,25 @@ def get_operator_classnames(cls) -> List[str]:
         return ["BashOperator"]
 
     def extract(self) -> Optional[TaskMetadata]:
+        collect_source = True
         if os.environ.get(
             "OPENLINEAGE_AIRFLOW_DISABLE_SOURCE_CODE", "True"
         ).lower() in ('true', '1', 't'):
-            return None
+            collect_source = False
```

Comment:
Could you show me exactly what you mean? I was trying to make this the same as how the [PythonExtractor does it](https://github.com/OpenLineage/OpenLineage/blob/main/integration/airflow/openlineage/airflow/extractors/python_extractor.py#L29)

Since they are so similar perhaps we should actually make the python and bash extractor be sub-classes of some parent but if so that might be better off in a different PR?

Reply:
Simple as that:
```
collect_source = os.environ.get(
    "OPENLINEAGE_AIRFLOW_DISABLE_SOURCE_CODE", "True"
     ).lower() not in ('true', '1', 't')
```

`PythonExtractor` has indeed same code, should be changed by the way.

Reply:
ok, what I did for now is simplify it in both places per your suggestion. I'll leave any potential merge of that code into a single parent class for a future PR.